src/standalone/Vegetation/Canopy.jl

```diff
@@ -476,6 +476,107 @@ function CanopyModel{FT}(;
     return CanopyModel{FT, typeof.(args)...}(args...)
 end
 
+"""
+    function CanopyModel{FT}(
+        domain::Union{
+            ClimaLand.Domains.Point,
+            ClimaLand.Domains.Plane,
+            ClimaLand.Domains.SphericalSurface,
+        },
+        forcing::NamedTuple,
+        LAI::Union{Function, AbstractTimeVaryingInput},
+        parameters::SharedCanopyParameters{FT, PSE};
+        prognostic_land_components = (:canopy,),
+        autotrophic_respiration = AutotrophicRespirationModel{FT}(),
+        radiative_transfer = TwoStreamModel{FT}(domain),
+        photosynthesis = FarquharModel{FT}(domain),
+        conductance = MedlynConductanceModel{FT}(domain),
+        hydraulics = PlantHydraulicsModel{FT}(domain, forcing),
+        energy = BigLeafEnergyModel{FT}(),
+        sif = Lee2015SIFModel{FT}(),
+    ) where {FT, PSE}
+
+Creates a CanopyModel with the provided domain, forcing, and parameters.
+
+Defaults are provided for each canopy component model, which can be overridden
+by passing in a different instance of that type of model. Default parameters are also provided
+for each canopy component, and can be changed with keyword arguments. Please see the documentation
+of each component model for details on the default parameters.
+
+The required argument `forcing` should be a NamedTuple with the following field:
+- `atmos`: a PrescribedAtmosphere or CoupledAtmosphere object
+- `radiation`: a PrescribedRadiativeFluxes or CoupledRadiativeFluxes object
+- `ground`: a PrescribedGroundConditions or PrognosticGroundConditions object
+
+The required argument `LAI` should be a function or ClimaUtilities TimeVaryingInput for leaf area index.
+
+When running the canopy model in standalone mode, set `prognostic_land_components = (:canopy,)`,
+while for running integrated land models, this should be a list of the individual models.
+This value of this argument must be the same across all components in the land model.
+"""
+function CanopyModel{FT}(
+    domain::Union{
+        ClimaLand.Domains.Point,
+        ClimaLand.Domains.Plane,
+        ClimaLand.Domains.SphericalSurface,
+    },
+    forcing::NamedTuple,
+    LAI::Union{Function, AbstractTimeVaryingInput},
+    parameters::SharedCanopyParameters{FT, PSE};
+    prognostic_land_components = (:canopy,),
+    autotrophic_respiration = AutotrophicRespirationModel{FT}(),
+    radiative_transfer = TwoStreamModel{FT}(domain),
+    photosynthesis = FarquharModel{FT}(domain),
+    conductance = MedlynConductanceModel{FT}(domain),
+    hydraulics = PlantHydraulicsModel{FT}(domain, LAI),
+    energy = BigLeafEnergyModel{FT}(),
+    sif = Lee2015SIFModel{FT}(),
+) where {FT, PSE}
+    (; atmos, radiation, ground) = forcing
```

Comment:
Right now, `ground` is not part of the `drivers` for Canopy if it is prognostic. https://github.com/CliMA/ClimaLand.jl/blob/f10d8f417484287ad2a5c425c1b74cbd7aa641ea/src/standalone/Vegetation/canopy_boundary_fluxes.jl#L86

Do we want to maintain that or always include it?

Reply:
ok! good point. Ill open a PR to do that once this is merged.